repos/spack_repo/builtin/packages/py_meson_python/package.py

```diff
@@ -33,26 +34,30 @@ class PyMesonPython(PythonPackage):
         deprecated=True,
     )
 
-    depends_on("c", type="build")  # generated
-
-    depends_on("py-colorama", when="platform=windows", type=("build", "run"))
-    depends_on("meson@0.63.3:", when="@0.11:", type=("build", "run"))
-    depends_on("meson@0.63:", when="@0.9:0.10", type=("build", "run"))
-    depends_on("meson@0.62:", type=("build", "run"))
-    depends_on("py-packaging@19:", when="@0.16:", type=("build", "run"))
-    depends_on("py-pyproject-metadata@0.7.1:", when="@0.13:", type=("build", "run"))
-    depends_on("py-pyproject-metadata@0.6.1:", when="@0.12:", type=("build", "run"))
-    depends_on("py-pyproject-metadata@0.5:", type=("build", "run"))
-    depends_on("py-tomli@1:", when="@0.11: ^python@:3.10", type=("build", "run"))
-    depends_on("py-tomli@1:", when="@:0.10", type=("build", "run"))
-    depends_on("py-setuptools@60:", when="@0.13 ^python@3.12:", type=("build", "run"))
-    depends_on("ninja", type=("build", "run"))
+    depends_on("c", type="build")
+
+    with default_args(type=("build", "run")):
+        depends_on("ninja", type=("build", "run"))
```

Comment:
Is this ninja entry supposed to still pass type=

Reply:
Oops, no, that's unnecessary, but also won't break anything. Can probably just fix that next time someone notices.